### ISSUE TITLE
Repeating events: Updating future occurences only

### ISSUE THREAD

**Issue by Faark** [SimpleMobileTools/Simple-Calendar#1103]
A weekly event of mine changed it's time. So I opened the EventActivity, set the new time and clicked save.

First surprise: I expected to have the same options I'm familiar with from deleting events... specifically "single", "future" and "all". "future" was missing.

Second surprise: After choosing "all", occurrences before the selected date disappeared. Including some yet to come. I expected (and accepted) them all to just get a different starting time, but at least stay on the same day as reminder. I certainly didn't expect the selected occurrence to become the new starting date of the reoccurring event.

In summary: I propose giving updating events the same 3 options we have when deleting. "Update this and future events" option that doesn't change occurrences prior to the selected one seems somewhat easy to implement. I understand updating events prior the selected might not. Can't see an easy way to implement it, at least. In that case what about just dropping the "update all" option, since IMO it hardly acts the way a user would expect, anyway.

Thanks for your work and such an awesome calendar.

**Comment by tibbi** [comment]
ye updating repeating events is currently handled pretty bad, but this is a duplicate of #981 . Thanks
